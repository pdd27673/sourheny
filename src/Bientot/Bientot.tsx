import React from "react";
import Typewriter from "typewriter-effect";
import "./Bientot.css";

type Props = {};

const Bientot = (props: Props) => {
    return (
        <div className="container">
            <h1 className="title">SOURHENY</h1>
            <h1 className="comingSoon">
                <Typewriter
                    options={{
                        strings: [
                            "BientÃ´t disponible...",
                            "Coming Soon...",
                        ],
                        autoStart: true,
                        loop: true,

                    }}
                />
            </h1>
        </div>
    );
};

export default Bientot;
