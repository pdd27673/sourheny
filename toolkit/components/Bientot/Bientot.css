.comingSoon {
    color: rgb(100, 100, 100);
    font-size: xx-large;
}

.title {
    color: black;
    font-size: xx-large;
}

.container {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    height: 100vh;
    width: 100vw;
    background-image: linear-gradient( 95.2deg, rgba(173,252,234,1) 26.8%, rgba(192,229,246,1) 64% );
}