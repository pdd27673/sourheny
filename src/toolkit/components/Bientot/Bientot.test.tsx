import React from "react";
import { expect, it } from "vitest";
import renderer from 'react-test-renderer';

import Bientot from "./Bientot";

it("renders correctly", () => {
    const result = renderer.create(<Bientot />).toJSON();
    expect(result).toMatchSnapshot();
});
