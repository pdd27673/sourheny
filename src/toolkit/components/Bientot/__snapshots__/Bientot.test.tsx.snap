// Vitest Snapshot v1

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <h1
    className="title"
  >
    SOURHENY
  </h1>
  <h1
    className="comingSoon"
  >
    <div
      className="Typewriter"
      data-testid="typewriter-wrapper"
    />
  </h1>
</div>
`;
